src/rust/multi.simplyhentai/res/settings.json

```diff
@@ -0,0 +1,16 @@
+[
+  {
+    "type": "group",
+    "title": "Settings",
+    "items": [
+      {
+        "type": "select",
+        "key": "image_quality",
+        "title": "Image Quality",
+        "values": ["full", "small_thumb", "thumb", "giant_thumb"],
+        "titles": ["Full", "Small Thumb", "Thumb", "Giant Thumb"],
```

Comment:
I don't think average users will understand why you're talking about thumbs. You should make this more readable, like Full, Small, Regular, Giant or something

Reply:
sure ok